Makefile

```diff
@@ -26,8 +26,8 @@ fmt-%:
 	cd $(subst :,/,$*); go fmt ./...
 
 vet-%:
-	@if [ "$(PKG)" = "git" ]; then \
-		cd $(PKG); make vet ;\
+	@if [ "$(PKG)" = "git:libgit2" ]; then \
```

Comment:
Maybe just some comments around this, as some folks (such as myself) may not understand this syntax without first looking it up.